Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -605,7 +700,32 @@ define([
         //>>includeEnd('debug');
 
         var url = buildImageUrl(this, x, y, level);
-        return ImageryProvider.loadImage(this, url);
+        if (!defined(this._requestNewToken)) {
+            return ImageryProvider.loadImage(this, url);
+        } else {
+            let that = this;
+            let tokenRetries = 1;
+            function loadImageWithToken () {
+                return loadImageViaBlob(url).otherwise(function(requestErrorEvent) {
```

Comment:
Oops one more: you need the `throttleRequestByServer` call.  See `ImageryProvider.loadImage`.

Reply:
Ah right.

ae96af9afe12d1934405375d1a97d0fe5d642f09